Library/Formula/khal.rb

```diff
@@ -89,14 +89,15 @@ class Khal < Formula
   end
 
   def install
-    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python2.7/site-packages"
+    version = Language::Python.major_minor_version "python"
+    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python#{version}/site-packages"
```

Comment:
Nothing really.
I had some strange runtime error, so I wanted to try if the same error comes up with python3 (it did as it was a config issue on my end).
however, python2 support might be dropped in future, so i copied this snipped from my vdirsyncher.rb which uses python3.